experimental/ner_wikiner_accelerated_with_nebullvm/configs/default.cfg

```diff
@@ -0,0 +1,134 @@
+[paths]
+train = ""
+dev = ""
+raw = null
+init_tok2vec = null
+vectors = null
+
+[system]
+gpu_allocator = null
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","ner"]
```

Comment:
It may be confusing to users that the `transformer` component is named `tok2vec`, since there's a separate `tok2vec` component. (Yes, the naming here is confusing on our end.)